{
  "type": "class",
  "name": "NumberInvalidException",
  "qualified_name": "net.minecraft.command.NumberInvalidException",
  "package": "net.minecraft.command",
  "modifiers": [],
  "annotations": [],
  "description": "",
  "inheritance": {
    "extends": null,
    "implements": [],
    "inheritance_tree": [
      "net.minecraft.command.CommandException"
    ]
  },
  "nested_classes": [],
  "fields": [],
  "constructors": [
    {
      "name": "NumberInvalidException",
      "modifiers": [],
      "parameters": [],
      "description": "",
      "deprecated": false
    },
    {
      "name": "NumberInvalidException",
      "modifiers": [],
      "parameters": [],
      "description": "",
      "deprecated": false
    }
  ],
  "methods": [
    {
      "name": "fillInStackTrace",
      "modifiers": [
        "public"
      ],
      "return_type": "",
      "parameters": [],
      "description": "",
      "deprecated": false
    }
  ],
  "enum_constants": [],
  "since": null,
  "deprecated": false,
  "source_file": "Frameworks/1.12.2/net/minecraft/command/NumberInvalidException.html"
}